JustSaying/JustSayingBus.cs

```diff
@@ -175,14 +173,20 @@ private IMessagePublisher GetActivePublisherForMessage(Message message)
             return publishersByTopic[topic];
         }
 
-        private async Task PublishAsync(IMessagePublisher publisher, Message message, int attemptCount = 0, CancellationToken cancellationToken = default)
+        private async Task PublishAsync(
+            IMessagePublisher publisher,
+            Message message,
+            PublishMetadata metadata,
+            int attemptCount = 0,
```

Comment:
Do any of the methods in this class ever call this method without specifying `attemptCount` or `cancellationToken`? If not, can make them non-optional so they don't accidentally get forgotten.